README.md

```diff
@@ -63,6 +63,41 @@ Note: x64 Ruby for Windows is known to have some compatibility issues.
 
 # Getting Started with Azure Resource Manager Usage (Preview)
 
+## Prerequisite
+
+In order to use the Azure SDK, you must supply the following values to the Azure SDK:
+
+* Tenant Id
+* Client Id
+* Subscription Id
+* Client Secret
+
+You could pass the above values in the following ways:
+
+### Option 1 - Environment Variables
+You can set the (above) values using the following environment variables:
+
+* AZURE_TENANT_ID
+* AZURE_CLIENT_ID
+* AZURE_SUBSCRIPTION_ID
+* AZURE_CLIENT_SECRET
+
+### Option 2 - Options Hash
+You can set the (above) values using the options hash:
```

Comment:
I think we need to give context on what the options hash is - for example "the options hash getting passed as parameter when constructing a client for the service/profile"

Reply:
Done